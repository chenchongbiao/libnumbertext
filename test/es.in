1001000000
1001001001000000
31000000
131000000
1031000000
10031000000
100031000000
1000031000000
21000000
121000000
1021000000
10021000000
100021000000
1000021000000
231231
531531
