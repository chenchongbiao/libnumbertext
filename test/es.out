mil un millones
mil un billones mil un millones
treinta y un millones
ciento treinta y un millones
mil treinta y un millones
diez mil treinta y un millones
cien mil treinta y un millones
un billón treinta y un millones
veintiún millones
ciento veintiún millones
mil veintiún millones
diez mil veintiún millones
cien mil veintiún millones
un billón veintiún millones
doscientos treinta y un mil doscientos treinta y uno
quinientos treinta y un mil quinientos treinta y uno
